content/en/openpnp/calibration/connect-to-machine/index.md

```diff
@@ -9,35 +9,86 @@ description: >
+
+11. Optional: in the `Format` drop-down, choose the `1280x720 10fps` setting
+
+12. Click the "Apply" button in the bottom right
+  {{< container-image path="images/Bottom Camera Apply.png" alt="Saving changes to the Bottom Camera Config" >}}
+
+## Top Camera Config
+
+13. Again, navigate to the `Machine Setup` tab
+14. Again, click the "Expand" checkbox if necessary
+15. Navigate to `Heads > ReferenceHead H1 > Cameras > OpenPnPCaptureCamera Top`
+  {{< container-image path="images/Top-camera-settings.png" alt="Finding the Top Camera Settings" >}}
+
+16. In the lower detail pane, switch to the `Device Settings` tab
+  {{< container-image path="images/Top-camera-device-settings.png" alt="Switching to the camera device settings" >}}
+
+17. In the `Device` drop-down, choose `PnP Bottom Camera (PnP Top Camera)`
+  {{< container-image path="images/Bottom-camera-select-device.png" alt="Selecting the correct device for the Bottom Camera" >}}
+
+18. Optional: in the `Format` drop-down, choose the `1280x720 10fps` setting
+
+19. Click the "Apply" button in the bottom right
+  {{< container-image path="images/Top Camera Apply.png" alt="Saving changes to the Top Camera Config" >}}
+
+20. To check that both cameras are working now, select your preferred view option in the top left `Cameras` section. The Camera feeds should update automatically with a very slight delay.
+  {{< container-image path="images/Both-cameras-working.png" alt="Both cameras showing live outputs." >}}
+
+## Connecting to the LumenPnP
 
-6. The power button should turn red, and OpenPnP has connected to your machine. You should also see both webcam feeds as shown below. (If you don't see any, or just one, under the `Cameras` section, select the `Show All Horizontal` option from the dropdown.)
+21. Click the green power button in the Machine Controls section of the UI to connect to your machine.
+  {{< container-image path="images/Screen Shot 2022-05-17 at 4.14.47 PM.png" alt="" >}}
 
-{{< container-image path="images/Screen Shot 2022-05-17 at 4.16.20 PM.png" alt="" >}}
+22. The power button should turn red, and OpenPnP has connected to your machine. You should also see both webcam feeds as shown below. (If you don't see any, or just one, under the `Cameras` section, select the `Show All Horizontal` option from the dropdown.)
+  {{< container-image path="images/Screen Shot 2022-05-17 at 4.16.20 PM.png" alt="" >}}
 
-7. Save your OpenPnP settings with File -> Save Configuration.
+23. While homing won't work properly until we finish calibrating the fiducial locations, you should now be able to manually "jog" your LumenPnP. Select 1mm or 10mm on the Distance slider, and then press the arrow buttons to move each axis of your machine. Again, since the machine isn't properly homed it may not move all the way to its boundaries. This is simply a test to make sure we can send it Gcode successfully.
```

Comment:
```suggestion
23. While homing won't work properly until you finish calibrating the fiducial locations, you should now be able to manually "jog" your LumenPnP. Select 1mm or 10mm on the Distance slider, and then press the arrow buttons to move each axis of your machine. Again, since the machine isn't properly homed it may not move all the way to its boundaries. This is simply a test to make sure that OpenPnP can send it Gcode successfully.
```